src/textUtils.c

```diff
@@ -53,13 +53,35 @@ size_t str_formatAdaAmount(uint64_t amount, char* out, size_t outSize)
 	return rawSize;
 }
 
+
+// TODO: This is valid only for mainnet
+static struct {
+	uint64_t startBlockNumber;
```

Comment:
Does this mean absolute slot number? There can be fewer blocks than slots.

Reply:
yep, that is absolute slot number. I did not want to confuse slots (which are usually meant per-epoch) with this absolute number